### PR TITLE
core: frontend: components: video-manager: Add reset stream settings button

### PR DESCRIPTION
**Pull request by patrickelectric**
Waiting for a new version of mavlink-camera-manager :ok: 

Signed-off-by: Patrick José Pereira <patrickelectric@gmail.com>

### CODE REVIEW

File: core/frontend/src/components/video-manager/VideoManager.vue [hunk 1]
```diff
@@ -33,6 +33,46 @@
...
+      @input="show_reset_dialog = false"
+    >
+      <v-card>
+        <v-card-title>
+          Reset video settings
+        </v-card-title>
+
+        <v-card-actions class="d-flex flex-column justify-space-around align-center pb-6">
+          <v-btn
+            @click="reset_settings"
```
Comment (patrickelectric) [inline core/frontend/src/components/video-manager/VideoManager.vue:66]: Method is missing

**Comment by joaoantoniocardoso** [comment]
It is lacking the "success" feedback for the user, otherwise, it looks good to me

**Comment by patrickelectric** [comment]
@joaoantoniocardoso, done.

**Review by joaoantoniocardoso — APPROVED** [review]
Awesome!

**Pull request merged**